packages/sdk/src/messages/NomadMessage.ts

```diff
@@ -450,13 +462,22 @@ export class NomadMessage<T extends NomadContext> {
   async replicaStatus(): Promise<ReplicaMessageStatus> {
     // backwards compatibility. Older replica versions returned a number,
     // newer versions return a hash
-    const root = (await this.replica.messages(this.leaf)) as string | number;
+    let root: string | number = await this.replica.messages(this.leaf);
+    root = root as string | number;
```

Comment:
is this necessary given the type is also defined above?

Reply:
yes, without this, it binds root's type as `string` 🙄